release_tools/versions.sh

```diff
@@ -1,5 +1,5 @@
-version=1.3.0
-previous_version=1.3.0
+version=1.3.0-alpha1
+previous_version=1.2.17
```

Comment:
Why do we want to check ABI compat. against the `1.2.x` series if we don't care about ABI compat. within the `1.3.x` series? :)

Reply:
It's an important part of release process to check API/ABI compatibility with the previous released version. We have an ABI/API compliance checker and we produce and publish reports where users can easily see which symbols were added, removed, or changed.